tests/e2e/support/constants.js

```diff
@@ -33,5 +33,4 @@ export const nodeTypes = {
 };
 
 export const boundaryEventSelector = '.main-paper ' +
-  '[data-type="processmaker.components.nodes.task.Shape"] + ' +
```

Comment:
After a pool is moved, many embedded objects appear between the task and the boundary event. We had to remove it.